doc/api/tls.md

```diff
@@ -896,6 +919,21 @@ changes:
     verified against the list of supplied CAs. An `'error'` event is emitted if
     verification fails; `err.code` contains the OpenSSL error code. **Default:**
     `true`.
+  * `pskCallback` {Function}
+     * hint: {Buffer} optional message sent from the server to help client
+       decide which identity to use during negotiation.
+     * Returns: {Object} in the format `{ psk: <string>, identity: <string> }`
+       where `psk` must be a hexadecimal string and `identity` must use UTF-8
+       encoding. Or null to stop the negotiation process.
+    When negotiating TLS-PSK (pre-shared keys), this function is called
+    with the identity hint provided by the server. If the client should
+    continue to negotiate PSK ciphers, the return value of this function must not
+    be null.
+    Note that PSK ciphers are disabled by default, and using TLS-PSK thus
+    requires explicitly specifying a cipher suite with the `ciphers` option.
```

Comment:
I suspect I could find the names of PSK ciphers through the `openssl` CLI, but I think our docs should tell people where to find them.